java.source.base/src/org/netbeans/api/java/source/TreeUtilities.java

```diff
@@ -1928,20 +1928,32 @@ public boolean isPartOfCompoundVariableDeclaration(@NonNull Tree tree) {
     }
 
     /**
-     * Check the tree has compile error in given collection of errors.
+     * Check the tree has compile error in given errors.
      *
      * @param tree : compilation tree
```

Comment:
Do not write `:` or other text separator between `@param` name and its content: the doclet generator will separate `@param` name and format using HTML; in this case, `:` will remain orphaned somewehere.

Reply:
Changes added